src/gui/windows/inventory_crafting.zig

```diff
@@ -2,9 +2,9 @@ const std = @import("std");
 
 const main = @import("main");
 const items = main.items;
-const BaseItem = items.BaseItem;
 const Inventory = items.Inventory;
 const ItemStack = items.ItemStack;
+const BaseItemIndex = items.BaseItemIndex;
```

Comment:
Why change the sorting order? It was alphabetical before.

Reply:
It is alphabetical but below parent symbol, I assume. That is very tricky to recognize as alphabetical, looks much more random. This is also additional effort to preserve manually. Changed tho.

Reply:
Maybe we should separate aliasing segment from importing segment to make it more organized? Then you would always sort by aliases, independently from import order. Imports would be sorted separately, again independent from each other.

Reply:
I think the aliases should all be directly below their parents, everything else sorted alphabetically.